packages/data-stores/src/domain-suggestions/index.ts

```diff
@@ -15,6 +15,7 @@ import type { DispatchFromMap, SelectFromMap } from '../mapped-types';
 import { controls } from '../wpcom-request-controls';
 
 export * from './types';
+export * from './constants';
```

Comment:
Scope creep?

Reply:
It's necessary for importing this line https://github.com/Automattic/wp-calypso/pull/50889/files#diff-81e5b2095be6643ef81b316eb251c80e5e29b24af18bbae45c019d70edd95ebaR6